index.ts

```diff
@@ -4,7 +4,7 @@ import cors from 'cors';
 import { graphqlHTTP } from 'express-graphql';
 import api from './server';
 import { schema, rootValue } from './server/graphql';
-import defaultQuery from './server/graphql/examples';
+import { timelineQuery } from './server/graphql/examples';
```

Comment:
We don't need to change this, the defaultQuery var should include all the queries

Reply:
@bonustrack you mean like an object? `defaultQuery.timeline`?

Reply:
No i mean just `defaultQuery` and change on server/graphql/examples.ts to be just a single default export with 2 queries, it should return something like this https://hub.snapshot.page/graphql?query=query%20Timeline%20%7B%0A%20%20timeline(first%3A%2020%2C%20skip%3A%200%2C%20spaces%3A%20%5B%22balancer%22%2C%20%22yam.eth%22%5D%2C%20state%3A%20%22closed%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%20%20start%0A%20%20%20%20timestamp%0A%20%20%20%20end%0A%20%20%20%20state%0A%20%20%20%20author%20%7B%0A%20%20%20%20%20%20address%0A%20%20%20%20%7D%0A%20%20%20%20space%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20members%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A%0Aquery%20Spaces%20%7B%0A%20%20spaces(first%3A%2020%2C%20skip%3A%200)%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%7D%0A%7D%0A

Reply:
https://hub.snapshot.page/graphql?query=query%20Timeline%20%7B%0A%20%20timeline(first%3A%2020%2C%20skip%3A%200%2C%20spaces%3A%20%5B%22balancer%22%2C%20%22yam.eth%22%5D%2C%20state%3A%20%22closed%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%20%20start%0A%20%20%20%20end%0A%20%20%20%20state%0A%20%20%20%20author%20%7B%0A%20%20%20%20%20%20address%0A%20%20%20%20%7D%0A%20%20%20%20space%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20members%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A%0Aquery%20Spaces%20%7B%0A%09spaces(first%3A%2010)%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%20%20about%0A%20%20%20%20private%0A%20%20%20%20members%0A%20%20%20%20strategies%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20params%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

Reply:
@bonustrack awesome 🚀 added that